src/OctoshiftCLI.Tests/bbs2gh/Commands/MigrateRepoCommandTests.cs

```diff
@@ -140,5 +150,58 @@ public async Task Skip_Migration_If_Target_Repo_Exists()
             // Assert
             _mockOctoLogger.Verify(m => m.LogWarning(It.IsAny<string>()), Times.Exactly(1));
         }
+
+        [Fact]
+        public async Task Hits_Bitbucket_With_The_Right_Options()
+        {
+            // Arrange
+            _mockBbsApiFactory.Setup(m => m.Create(It.IsAny<string>(), It.IsAny<string>(), It.IsAny<string>())).Returns(_mockBbsApi.Object);
+
+            _mockBbsApi.Setup(x => x.StartExport(BBS_PROJECT, BBS_REPO)).ReturnsAsync(BBS_EXPORT_ID);
+            _mockBbsApi.Setup(x => x.GetExport(BBS_EXPORT_ID)).ReturnsAsync(("COMPLETED", "The export is complete", 100));
+
+            // Act
+            var args = new MigrateRepoCommandArgs
+            {
+                BbsServerUrl = BBS_SERVER_URL,
+                BbsUsername = BBS_USERNAME,
+                BbsPassword = BBS_PASSWORD,
+                BbsProject = BBS_PROJECT,
+                BbsRepo = BBS_REPO
+            };
+            await _command.Invoke(args);
+
+            // Assert
+            _mockBbsApi.Verify(m => m.StartExport(
+                BBS_PROJECT,
+                BBS_REPO
+            ));
+
+        }
+
+        [Fact]
+        public async Task Throws_An_Error_If_Export_Fails()
+        {
+
+            // Arrange
+            _mockBbsApiFactory.Setup(m => m.Create(It.IsAny<string>(), It.IsAny<string>(), It.IsAny<string>())).Returns(_mockBbsApi.Object);
+
+            _mockBbsApi.Setup(x => x.StartExport(BBS_PROJECT, BBS_REPO)).ReturnsAsync(BBS_EXPORT_ID);
+            _mockBbsApi.Setup(x => x.GetExport(BBS_EXPORT_ID)).ReturnsAsync(("FAILED", "The export failed", 0));
+
+            // Act
+            var args = new MigrateRepoCommandArgs
+            {
+                BbsServerUrl = BBS_SERVER_URL,
+                BbsUsername = BBS_USERNAME,
+                BbsPassword = BBS_PASSWORD,
+                BbsProject = BBS_PROJECT,
+                BbsRepo = BBS_REPO,
+                Wait = true
+            };
+
+            // Assert
+            await Assert.ThrowsAsync<OctoshiftCliException>(() => _command.Invoke(args));
```

Comment:
nit: this totally works and you can totally leave it as is, but in case you want to make it probably a bit for `Fluent`, here's how you do it with `FluentAssertions`:
```C#
await _command.Invoking(x => x.Invoke(args)).Should().ThrowExactlyAsync<OctoshiftCliException>();
```